com.unity.visualeffectgraph/Runtime/Utilities/Playables/VisualEffectControl/VisualEffectControlTrack.cs

```diff
@@ -44,6 +50,45 @@ protected override void OnBeforeTrackSerialize()
             }
         }
 
+#if UNITY_EDITOR
+        protected override void OnCreateClip(TimelineClip clip)
+        {
+            base.OnCreateClip(clip);
+            if (clip.asset is VisualEffectControlClip vfxClip)
+            {
+                vfxClip.clipStart = clip.start;
+                vfxClip.clipEnd = clip.end;
+
+                if (m_CurrentVisualEffect != null)
+                {
+                    //Copy Seed settings
+                    vfxClip.startSeed = m_CurrentVisualEffect.startSeed;
+
+                    //Copy Prewarm settings
+                    vfxClip.prewarm.eventName = m_CurrentVisualEffect.initialEventName;
+                    if (m_CurrentVisualEffect.visualEffectAsset != null)
+                    {
+                        using var resourceObject = new SerializedObject(m_CurrentVisualEffect.visualEffectAsset);
+                        var prewarmDeltaTime = resourceObject.FindProperty("m_Infos.m_PreWarmDeltaTime");
+                        var prewarmStepCount = resourceObject.FindProperty("m_Infos.m_PreWarmStepCount");
+                        if (prewarmDeltaTime != null && prewarmStepCount != null)
+                        {
+                            if (prewarmStepCount.uintValue != 0u)
```

Comment:
You can merge these two `if`

Reply:
Fix at bf30d3ae887b9acdb15caa2e1d0377be06df43e0